shlr/meson.build

```diff
@@ -286,9 +291,10 @@ qnx_files = [
 qnx_inc = [platform_inc, include_directories('qnx/include')]
 
 librzqnx = static_library('rzqnx', qnx_files,
-  dependencies: [rz_socket_dep, rz_util_dep],
+  dependencies: [rz_util_dep, rz_socket_dep],
```

Comment:
Ok I actually found out that all these reordering was not needed, it was just my tcc plugin in Meson that was broken. However, I'd still keep this commit because it removes some unnecessary dependencies.